BackendApartmentReservation/BackendApartmentReservation/Database/Entities/Amenities/DbLivingPlaceAmenity.cs

```diff
@@ -9,8 +9,6 @@ public class DbLivingPlaceAmenity : AmenityBase
         [Key]
         public int Id { get; set; }
 
-        public DbHotelReservation Hotel { get; set; }
-
-        public DbApartment Apartment { get; set; }
+        public DbLivingPlaceReservation LivingPlaceReservation { get; set; }

```

Comment:
What is this class for? We have hotel amenity, we have apartment amenity, what is living place in this case?

Also, there is `DbLivingPlaceReservation` which seems to be doing the same thing

Reply:
I think in some scenarios Living Place is a combination of Apartment + Hotel. For example an employee is booking a Hotel for 2 days and then an Apartment for 5 days, because it became free. I'm not sure if this is the best solution, but maybe it can stay like this at least for now?

Reply:
Well, I dont remember having such requirement. Maybe we should not overengineer at first iterations?